lib/FundDistribution/FundDistributionFields/FundDistributionFieldsFinalContainer.js

```diff
@@ -28,6 +30,7 @@ const FundDistributionFieldsFinalContainer = ({
   totalAmount,
   validate,
   validateFundDistributionTotal,
+  shouldRefetchExpenseClassesOnFiscalYearChange,
```

Comment:
comment is the same

> why do you need one more flag? you already have fiscalYearId, why not to introduce one more use effect to trigger expense classes refetch for selected fund?

Reply:
Sorry, I forgot to push my local changes :)